File: NETMouse - .NET release/Examples/PascalABC.NET/Arrays_6.pas
{$reference ABCNET.dll}
uses ABCNET.Utils;
uses ABCNET.Extensions;
begin
  Arr.Rand(BaseU.ReadInteger('N:'), -19, 26).Println()
    .Where((x, i) -> i mod 2 <> 0).Aggregate((a, b) -> a * b).Println();
  BaseU.ReadChar();
end.

File: NETMouse - .NET release/Examples/C#/Arrays_6.cs
using ABCNET.Extensions;
using ABCNET.Utils;
using System.Linq;

namespace TestProgram
{
    internal class Program
    {
        private static void Main(string[] args)
        {
            Arr.Rand(Base.ReadInteger("N:"), -19, 26).Println()
                .Where((x, i) => i % 2 != 0).Aggregate((a, b) => a * b).Println();
        }
    }
}